Src/Support/Google.Apis.Auth/OAuth2/BearerToken.cs

```diff
@@ -55,6 +55,7 @@ public string GetAccessToken(HttpRequestMessage request)
         /// Thread-safe OAuth 2.0 method for accessing protected resources using an <c>access_token</c> query parameter
         /// as specified in http://tools.ietf.org/html/rfc6750#section-2.3.
         /// </summary>
+        [Obsolete("https://developers.google.com/identity/protocols/oauth2/index.html#4.-send-the-access-token-to-an-api.")]
```

Comment:
I'd probably put something in the XML comment as well, just to hammer the point home.